docker/utils.js

```diff
@@ -244,6 +244,38 @@ exports.readLighthouseReport = (folder, writeLog) => {
 	return [lhr, lhrSummary];
 };
 
+/**
+ * parse Artillery Report
+ * @param {string} folder - .lighthouseci folder
+ * @param {func} writeLog - logging method
+ */
+exports.readArtilleryReport = (folder, writeLog) => {
+	if (!fs.existsSync(folder)) {
+		console.log(
+			'ERROR => No Artillery report found'
+		);
+		return [null, null];
+	}
+
+	writeLog(`Reading Artillery report files`);
+
+	const atr = JSON.parse(
+		fs.readFileSync(`artilleryOut.json`).toString()
+	);
+
+	const atrSummary = {
+		timestamp: atr.aggregate.timestamp,
+		scenariosCreated: atr.aggregate.scenariosCreated,
+		scenariosCompleted: atr.aggregate.scenariosCompleted,
+		requestsCompleted: atr.aggregate.requestsCompleted,
+		rpsCount: atr.aggregate.rps.count,
+		latencyMedian: atr.aggregate.latency.median,
+		scenarioCount: JSON.stringify(atr.aggregate.scenarioCounts),
```

Comment:
why this?

Reply:
This is the summary based on the aggregate result from Artillery

Reply:
I mean why we have to JSON.strigify it?